build.assets/robotest_run_nightly.sh

```diff
@@ -25,15 +25,23 @@ readonly ROBOTEST_SCRIPT=$(mktemp -d)/runsuite.sh
 
 # number of environment variables are expected to be set
 # see https://github.com/gravitational/robotest/blob/master/suite/README.md
-export ROBOTEST_VERSION=${ROBOTEST_VERSION:-stable-gce}
+export ROBOTEST_VERSION=${ROBOTEST_VERSION:-uid-gid}
```

Comment:
Is the use of this variable correct? Looks like it might contain a `uid:gid` pair rather than a semver.

Reply:
Yeah, it is a poorly named tag in quay.io https://quay.io/repository/gravitational/robotest-suite?tab=tags.  As mentioned in your other comment, I should be going to semver shortly after a 2 year hiatus for the project.